openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js

```diff
@@ -99,12 +99,12 @@ describe('Tags page should work', () => {
       .parent()
       .should('have.class', 'activeCategory');
     cy.get('[data-testid="add-new-tag-button"]').should('be.visible').click();
-    cy.get('.tw-modal-container').should('be.visible');
+    cy.get('[data-testid="modal-container"]').should('exist');
```

Comment:
we must check for visible instead of exist, this check will pass even if it is not appear on the screen, let me know if you need any help around this